### PR TITLE
DLive service name edited

### PR DESCRIPTION
**Pull request by berksmbl**

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
You didn't update the version to reflect the changes

**Review by Bas950 — CHANGES_REQUESTED** [review]
bump version

**Comment by StrikerFRFX** [comment]
Why close...? I would have approved...

**Comment by berksmbl** [comment]
> Why close...? I would have approved...
I apologize for not noticing.

File: websites/D/DLive/presence.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ presence.on("UpdateData", async () => {
       ".info-line-left.flex-box .flex-column.flex-justify-center div"
     );
     streamer = document.querySelector(
-      "div.channel-header span.dlive-name span.overflow-ellipsis"
+      "div.channel-header span.DLive-name span.overflow-ellipsis"
```
Comment (StrikerFRFX) [inline websites/D/DLive/presence.ts:42]: Upload proof of working

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]

**Comment by StrikerFRFX** [comment]
Oh man. Don't delete the presence.ts file, restore that. Undo the change on `Line 42` and make it back to what it was.

**Pull request closed without merging**